File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    removeDuplicates(nums) {
        let p1 = 1;
        for (let p2 = 1; p2 < nums.length; p2++) {
            if(nums[p2]!= nums[p1-1]){
                nums[p1]=nums[p2];
                p1++;
            }
        }
        return p1;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int n=nums.size();
        unordered_set<int> visited;
        unordered_set<int> s;
        for(int i=0;i<n;i++){
            s.insert(nums[i]);
        }
        int ans=0;
        for(int i=0;i<n;i++){
            if(!visited.count(nums[i]) && !s.count(nums[i]-1)){
                visited.insert(nums[i]);
                int c=1;
                for(int j=nums[i]+1;s.count(j);j++){
                    c++;
                    visited.insert(j);
                }
                ans=max(ans,c);
            }
        }
        return ans;
    }
};
